### PR TITLE
K8s helm extending configs

### PR DESCRIPTION
**Pull request by dsever**
@ptrovatelli 

Related to https://github.com/DefectDojo/django-DefectDojo/pull/2584 

Additional to previous PR test for pqsql replicas  and test for extra configs have been added

Old PR can can be closed.

### CODE REVIEW

**Review by ptrovatelli — APPROVED** [review]

**Comment by madchap** [comment]
fyi @ccojocar

**Comment by dsever** [comment]
@madchap,  after we did two days ado some changes inside of travis files i need to reflect the same into this PR `or tests will fail :/

**Comment by dsever** [comment]
> @madchap, after we did two days ado some changes inside of travis files i need to reflect the same into this PR `or tests will fail :/

Fixed

File: helm/defectdojo/templates/django-deployment.yaml [hunk 1]
```diff
@@ -71,6 +74,7 @@ spec:
               name: {{ .Values.mysql.existingSecret }}
               key: mysql-password
               {{- end }}
+<<<<<<< HEAD
```
Comment (ptrovatelli) [inline helm/defectdojo/templates/django-deployment.yaml]: this file seems part of an unfinished merge and will break the deployment

**Review by ptrovatelli — CHANGES_REQUESTED** [review]

**Comment by ptrovatelli** [comment]
@dsever sorry to insist, but are those commits on dev branch? if yes, you should rebase rather than picking commits. It makes the review much easier 
as per the PULL_REQUEST_TEMPLATE.md, you only need to: 

this first part is just to "erase" the commits you just picked/merged (assuming your clone is the remote `myOrigin` and the main github remote is `origin`;  check the remotes aliases with `git remote -v` )
```
git checkout k8s-helm-extending-configs
git rebase -i origin/dev
```
a file should open, replace in it "pick" by "drop" to erase the commits

Check the result with `git log`. You should be in the same state as yesterday

Then follow the PULL_REQUEST_TEMPLATE.md  :
`
git rebase dev k8s-helm-extending-configs
`

In case of conflict: 
```
 git mergetool
 git rebase --continue
 ```

Check the result with `git log`: again you should see the exact same commits as yesterday, just now they are placed after dev (except if you resolved conflicts)

When everything's fine on your local branch, force push to your `myOrigin` remote: 
```
git push myOrigin --force-with-lease
```

To cancel everything: 
```
git rebase --abort
```

thanks!

**Review by ccojocar — APPROVED** [review]
LGTM

**Review by madchap — APPROVED** [review]

**Comment by madchap** [comment]
lgtm too. Squash and merge anyways. @ptrovatelli you want to have a last look? @alles-klar ?

**Comment by dsever** [comment]
Please take a look https://github.com/DefectDojo/django-DefectDojo/pull/2847

It is squashed

**Comment by ptrovatelli** [comment]
https://github.com/DefectDojo/django-DefectDojo/pull/2847 is easier to review; closing this one

**Pull request closed without merging**